### PR TITLE
Added content-property-no-static-value

### PR DESCRIPTION
**Pull request by Uncleseneca**

### CODE REVIEW

File: src/rules/no-css-generated-content/__tests__/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+testRule(rule, {
+  ruleName,
+  config: [true],
+
+  accept: [
+    {
+      code: ".foo::after { content: ''; }",
+    },
+    {
+      code: '.foo::after { content: ""; }',
```
Comment (YozhikM) [inline src/rules/no-css-generated-content/__tests__/index.js]: Add to accepted tests next, please
```js
{
  code: "a { }",
},
{
  code: ".foo:after { content: ''; }", 
}
```

File: src/rules/no-css-generated-content/__tests__/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    {
+      code: '.bar::before { content: attr(aria-label); }',
+    },
+  ],
+
+  reject: [
+    {
+      code: '.foo::before { content: "bar"; }',
```
Comment (YozhikM) [inline src/rules/no-css-generated-content/__tests__/index.js]: And to rejected
```js
{
  code: ".foo:before, .bar { content: ''; }", 
},
```

**Comment by YozhikM** [comment]
@Uncleseneca 
Thanks for PR! I've requested some changes to make the rule even better.

File: src/rules/no-css-generated-content/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# no-css-generated-content-in-pseudo-elements
```
Comment (YozhikM) [inline src/rules/no-css-generated-content/README.md]: Maybe just `no-css-generated-content`?

File: src/rules/index.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import noCssGeneratedContent from './no-css-generated-content';
```
Comment (YozhikM) [inline src/rules/index.js]: Also, the rule is to add to main README

File: src/rules/no-css-generated-content/README.md (hunk 3 above)
Comment (evilebottnawi) [reply src/rules/no-css-generated-content/README.md]: `content-property-no-static-value`?

Comment (YozhikM) [reply src/rules/no-css-generated-content/README.md]: @evilebottnawi I like it.

**Comment by Uncleseneca** [comment]
Thanks for feedback!
I made changes according to review

**Pull request merged**